From: brian@murphy.dk
Subject: [PATCH] USB: usbfs: remove debug message

[PATCH] USB: usbfs: remove debug message

	this removes a warning message which only contains
data which is in any case returned to user space. This message
is especially annoying when polling with short timeouts, filling
up the log files and polluting the console. As agreed with
David Brownell.


Signed-off-by: Brian Murphy <brian@murphy.dk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/core/devio.c |    5 +----
 1 files changed, 1 insertion(+), 4 deletions(-)


diff -Nru a/drivers/usb/core/devio.c b/drivers/usb/core/devio.c
--- a/drivers/usb/core/devio.c	2005-03-08 16:42:53 -08:00
+++ b/drivers/usb/core/devio.c	2005-03-08 16:42:53 -08:00
@@ -673,11 +673,8 @@
 		usb_lock_device(dev);
 	}
 	kfree(tbuf);
-	if (i < 0) {
-		dev_warn(&dev->dev, "usbfs: USBDEVFS_BULK failed "
-			 "ep 0x%x len %u ret %d\n", bulk.ep, bulk.len, i);
+	if (i < 0)
 		return i;
-	}
 	return len2;
 }
 
